src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -2629,6 +2629,12 @@ bool CacheManager::Init(const std::string& file_name,
       LOG4CXX_DEBUG(logger_,
                     "Check if snapshot valid: " << std::boolalpha << result);
 
+      result &= UnwrapAppPolicies(pt_->policy_table.app_policies_section.apps);
+      if (!result) {
+        LOG4CXX_ERROR(logger_, "Cannot unwrap application policies");
+        return result;
```

Comment:
Should we really return if we cannot unwrap app policies?

What if the snapshot was invalid? 
Do we want to skip reporting the snapshot errors and doing `ex_backup_->RemoveDB();`?

Reply:
If Unwrap Policies was unsuccessful, it means, that preloaded_pt is wrong, and we should not start sdl.  The second option it to make UnwrapAppPolicies function void, and do not care about it's result.